src/contrib/mlir/lowerer.cpp

```diff
@@ -59,6 +61,13 @@ namespace
 
 #include "op_lowerers.inc"
 
+    // Helpers
+    template <typename RedOp>
+    void lowerIndexReduction(Operation* op,
```

Comment:
(General comment, just picking on this line at random.) nGraph coding standards would call for `lower_index_reduction` here. Is this an MLIR style thing? If so, which style guidelines (MLIR or nGraph) should take precedence? (If not, can we switch things over to nGraph style?)

Reply:
The idea was to keep nGraph style in the "entry point" files (i.e., compiler.hpp/.cpp) so that we keep nGraph style consistent in external APIs, and follow MLIR style on the more internal files (such us this one) that heavily use MLIR APIs. That would reduce the mix of nGraph/MLIR styles since we cannot control MLIR style. However, I think we are not being very consistent. I'll follow up with a commit to fix the style and add a comment about the style at the beginning of the file if there is no problem with this approach.

Reply:
I'd prefer to use nGraph style in both places, but as long as you're consistent about it I don't see it as a major issue.

Reply:
Thanks, Adam. I leave it like this for now and open this question for discussion later on.